roles/lib_openshift/src/lib/rolebinding.py

```diff
@@ -0,0 +1,276 @@
+# pylint: skip-file
+
+# pylint: disable=too-many-instance-attributes
+class RoleBindingConfig(object):
+    ''' Handle route options '''
+    # pylint: disable=too-many-arguments
+    def __init__(self,
+                 sname,
```

Comment:
I think at one point importing * from ansible had a global 'name' conflict and was causing linters to break.  Will update to name since we no longer import *.